src/core-common/src/main/java/org/apache/kylin/common/util/Unsafe.java

```diff
@@ -29,6 +29,7 @@
 import org.apache.commons.lang.StringUtils;
 
 import lombok.extern.slf4j.Slf4j;
+import org.apache.kylin.common.KylinConfigBase;
```

Comment:
this import is weird, KylinConfigBase is an instance class, and KylinConfigBase can use Unsafe util, but Unsafe should not depend on KylinConfigBase.

Reply:
Have removed KylinConfigBase import from Unsafe.java